audioread/ffdec.py

```diff
@@ -231,9 +231,9 @@ def _parse_info(self, s):
             if mode == 'stereo':
                 self.channels = 2
             else:
-                match = re.match(r'(\d+) ', mode)
-                if match:
-                    self.channels = int(match.group(1))
+                cmatch = re.match(r'(\d).?(\d)?', mode)
```

Comment:
Awesome; thank you!

In the future, we will all listen to 12256-channel music. :smiley: